recipe/meta.yaml

```diff
@@ -1,32 +1,24 @@
 {% set name = "setuptools" %}
-{% set version = "75.8.0" %}
-{% set checksum = "c5afc8f407c626b8313a86e10311dd3f661c6cd9c09d4bf8c15c0e11f9f2b0e6" %}
+{% set version = "78.1.1" %}
+{% set checksum = "fcc17fd9cd898242f6b4adfaca46137a9edef687f43e6f78469692a5e70d851d" %}
 
-# make sure to set CONDA_ADD_PIP_AS_PYTHON_DEPENDENCY=0 environ-variable before building it
+# make sure to set CONDA_ADD_PIP_AS_PYTHON_DEPENDENCY=0 environ-variable before building it locally
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
   sha256: {{ checksum }}
-  patches:
-    # Modify setuptools to fail if used in conda build (encourage people to add all deps in meta.yaml).
-    - patches/0002-disable-downloads-inside-conda-build.patch
-    # distutils patches from python-feedstock
-    - patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
```

Comment:
Can you add back patch 0021? It is still required for packages built using distutils msvc.

Reply:
Already did. Lol

Reply:
Done! Lol I was already doing it but I didn't push it up fast enough. Lol